_sass/_hero.scss

```diff
@@ -0,0 +1,27 @@
+.hero {
+  align-items: center;
+  background: $ember-orange;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: center;
+  width: 100vw;
```

Comment:
Yeah, that doesn't need to happen in this PR. I haven't used Jekyll in a long time, but it looks like Proteus has it built-in. 

https://github.com/thoughtbot/proteus-jekyll/blob/master/Gemfile